### PR TITLE
Update links to implementations

### PR DESCRIPTION
**Pull request by dyladan**

### CODE REVIEW

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -37,16 +37,17 @@ available.
...
+3. [JavaScript](https://github.com/open-telemetry/opentelemetry-js/blob/4f48357b7d1dea246a67ef1a0e071fc64aa10e26/packages/opentelemetry-core/src/context/propagation/HttpTraceContext.ts)
+4. [Java](https://github.com/open-telemetry/opentelemetry-java/blob/0b21928db5085fe3deff6dfd8d28bd49dcde9e71/api/src/main/java/io/opentelemetry/trace/propagation/HttpTraceContext.java)
+5. [Python](https://github.com/open-telemetry/opentelemetry-python/blob/602d42a45f6a4684342b298c5a7c4dab680e301a/opentelemetry-api/src/opentelemetry/context/propagation/tracecontexthttptextformat.py)
+6. [Ruby](https://github.com/open-telemetry/opentelemetry-ruby/blob/bd6baeebb2fe65ed22e9ad33920387ce0ee8870f/api/lib/opentelemetry/distributed_context/propagation/trace_parent.rb)
+
+Open Telemetry implementations are in active development for C++, Erlang, and PHP. In addition, [Open Census](https://opencensus.io) has an implementation for [Erlang](https://github.com/census-instrumentation/opencensus-erlang/blob/b3ab781b060b15a3cacbf43717c3aeb0c90c4a08/src/oc_propagation_http_tracecontext.erl).
```
Comment (mtwo) [inline README.md:50]:
```suggestion
OpenTelemetry is the next version of OpenCensus. OpenTelemetry implementations are in active development for C++, Erlang, and PHP. In addition, [Open Census](https://opencensus.io) has an implementation for [Erlang](https://github.com/census-instrumentation/opencensus-erlang/blob/b3ab781b060b15a3cacbf43717c3aeb0c90c4a08/src/oc_propagation_http_tracecontext.erl) and the languages listed above.
```

**Review by mtwo — CHANGES_REQUESTED** [review]
Added an explanation that the OpenTelemetry implementations replace the OpenCensus ones. Feel free to modify.

File: README.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply README.md:50]: I don't think it's relevant to this context.

Comment (dyladan) [reply README.md:50]: It's also not really accurate as OpenTelemetry is a replacement for OpenCensus and OpenTracing, and is not really the next version of either.

**Comment by dyladan** [comment]
@mtwo re-requested review. if our comments don't satisfy your concern pls let me know.

**Review by mtwo — APPROVED** [review]

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**